src/routes/travelRoute.js

```diff
@@ -44,4 +40,7 @@ route.patch(
 // Most travelled to destinations
 route.get('/most', getToken, verifyToken, mostTravelledDest);
 
+// handles editing of user's pending request
+route.put('/travel/pending', getToken, verifyToken, validateTravelRequest, validateResult, userCanEditOpenRequest);
```

Comment:
Why use PUT instead of PATCH for your update?
Also the endpoint is really not descriptive, and I also expect the travel ID of the travel request that a requester wants to edit to be part of the endpoint as a param.
I will suggest `/travel/update_request/:travel_id` as the endpoint for this feature.

Reply:
@chuksjoe  I appreciate your review

The requester may want to update more than 1 property. hence, the reason for using PUT. 

I assume the requester can have plenty requests on his or her dashboard, with an edit button attached to it. It is similar to recreating a trip. I think the endpoint without a param is fine.